workflows/emulation/train_emulator.py

```diff
@@ -0,0 +1,279 @@
+import argparse
```

Comment:
Script to train an "all-physics" emulator using the subset data.  Saves the model and tensorboard training info to GCS.  Model is loadable using the `AllPhysicsEmulator` Predictor class for use in prognostic runs.